inst/rmarkdown/templates/nutrition/skeleton/skeleton.Rmd

```diff
@@ -20,6 +20,13 @@ library(anthro) # for the WHO Child Growth Standards.
 
 **Introduction:** ...
```

Comment:
mmmm i think you might be on an old version of nutrition, let me double check

Reply:
yeap definitiely an old version of nutrition! Maybe it will add without too much trouble though

Reply:
fixed it